app/code/ElTrubetskaia/AskQuestion/view/adminhtml/layout/askquestion_page_index.xml

```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0"?>
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <head>
+        <css src="ElTrubetskaia_AskQuestion::css/customClasses.css"/>
```

Comment:
Don't include custom styles, add your styles to: ...<Namespace>_<Module>/web/css/source/_module.less

Reply:
If it's styles for adminhtml - should create a new theme for AdminPanel and then in app/design/adminhtml/.../web/css/source/_module.less apply new styles?

Reply:
sorry, didn't mentioned that it's admin. Ok